Mathlib/Geometry/Manifold/AnalyticManifold.lean

```diff
@@ -166,27 +111,78 @@ theorem symm_trans_mem_analyticGroupoid (e : PartialHomeomorph M H) :
 instance : ClosedUnderRestriction (analyticGroupoid I) :=
   (closedUnderRestriction_iff_id_le _).mpr
     (by
-      rw [StructureGroupoid.le_iff]
+      apply StructureGroupoid.le_iff.mpr
```

Comment:
Why did you revert this? If you want to golf that proof, please combine the last two lines as in
```
      rw [StructureGroupoid.le_iff]
      rintro e ⟨s, hs, hes⟩
      exact (analyticGroupoid I).mem_of_eqOnSource' _ _ (ofSet_mem_analyticGroupoid I hs) hes)  
```
(or leave it be, that is also fine).

Reply:
Done. Any revert was accidental.